libraries/disp/viewers/helpers/scalecontrol.h

```diff
@@ -0,0 +1,271 @@
+// QT INCLUDES
+//=============================================================================================================
+
+#include <QWidget>
+
+//=============================================================================================================
+// EIGEN INCLUDES
+//=============================================================================================================
+
+//=============================================================================================================
+// FORWARD DECLARATIONS
+//=============================================================================================================
+
+namespace Ui {
+    class ScaleControlWidget;
+}
+
+//=============================================================================================================
+// DEFINE NAMESPACE DISPLIB
+//=============================================================================================================
+
+namespace DISPLIB {
+
+//=============================================================================================================
+/**
+ * DECLARE CLASS ScaleControl
+ *
+ * @brief The ScaleControl class packs together a QLable, a DoubleSpinbox and a Slider with the necessary facilities for the
+ * interaction between them.
+ */
+class DISPSHARED_EXPORT ScaleControl : public QWidget
+{
+    Q_OBJECT
+
+public:
+
+    //=========================================================================================================
+    /**
+     * Constructs a ScaleControl object who's Qlabel elemennt will be set.
+     *
+     * @param [in] label    text for the Qlabel
+     */
+    ScaleControl(const char* label);
+
+    //=========================================================================================================
+    /**
+     * Constructs a ScaleControl object who's Qlabel elemennt will be set, and sets the parent of the underlyning
+     * QWidget.
+     *
+     * @param [in] label    text for the Qlabel
+     * @param [in] parent   parent of widget
+     */
+    ScaleControl(const char* label, QWidget* parent);
+
+    //=========================================================================================================
+    /**
+     * Constructs a ScaleControl object who's Qlabel elemennt will be set, and sets the parent of the underlyning
+     * QWidget. The min and max values for the control will also be initialized.
+     *
+     * @param [in] label    text for the Qlabel
+     * @param [in] parent   parent of widget
+     * @param [in] min      min value of the control.
+     * @param [in] max      max value of the control.
+     */
+     ScaleControl(const char* label, QWidget* parent, double min, double max);
+
+    //=========================================================================================================
+     /**
+     * getUI Retunr a pointer to the GUI of the ScaleControl.
+     * @return Pointer to the ScaleControlWidget.

```

Comment:
add line between function doc and return doc.
remove function name from function doc
Retunr -> Return